src/current/v24.2/logging-best-practices.md

```diff
@@ -24,6 +24,26 @@ Unstructured logs are more free-form and human-readable compared to machine-proc
 
 General Troubleshooting: They are useful for capturing a wide range of information that may not fit into a predefined structure, such as error messages or stack traces. Events not documented on [Notable Events]({% link {{ page.version.version }}/eventlog.md %}) will have an unstructured format in log messages.
 
+## Storage considerations for file sinks
+
+[Log sinks]({% link {{ page.version.version }}/configure-logs.md %}#configure-log-sinks) route events from specified [logging channels]({% link {{ page.version.version }}/logging.md %}#logging-channels) to destinations outside CockroachDB. These destinations currently include [file sinks]({% link {{ page.version.version }}/configure-logs.md %}#output-to-files), network sinks ([Fluentd-compatible]({% link {{ page.version.version }}/configure-logs.md %}#output-to-fluentd-compatible-network-collectors) servers and [HTTP]({% link {{ page.version.version }}/configure-logs.md %}#output-to-http-network-collectors) servers), and the [standard error stream (`stderr`)]({% link {{ page.version.version }}/configure-logs.md %}#output-to-stderr).
+
+With a file sink, when provisioning [storage]({% link {{ page.version.version }}/recommended-production-settings.md %}#storage) for your CockroachDB cluster, decide [where the log files will be stored]({% link {{ page.version.version }}/configure-logs.md %}#logging-directory): either the same volume as the main data store or on a separate volume from the main data store. To determine this, measure and consider the following factors:
+
+- How [IO intensive]({% link {{ page.version.version }}/recommended-production-settings.md %}#disk-i-o) is your workload.
```

Comment:
how can users monitor this? Perhaps we should link here: https://deploy-preview-19171--cockroachdb-docs.netlify.app/docs/v24.3/common-issues-to-monitor#disk-iops